lib/challenge.js

```diff
@@ -51,6 +67,14 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
+some_pet(name,species,breed,noise){
+  [
+name: oreo
+species: dog
+breed:Rottweiler
+noise: bark
+  ]
+}
```

Comment:
This is like above, but with the addition of square (array) brackets. Things it seems you intended as strings (`oreo`, `dog`, etc.) are actually references to variables that don't exist because the required quotation marks are missing.